addons/block_code/ui/util.gd

```diff
@@ -50,7 +50,7 @@ static func instantiate_block_by_name(block_name: StringName) -> Block:
 	return instantiate_block(block_definition)
 
 
-static func _get_subclasses(_class_name: String) -> Array[String]:
+static func _get_builtin_subclasses(_class_name: String) -> Array[String]:
```

Comment:
Nitpick. These aren't subclasses (classes that inherit this class), they're parent or inherited classes (what this class inherits from).

Reply:
Oh of course! Good catch, I'll rename the function.